functional_requirements.md

```diff
@@ -0,0 +1,55 @@
+# Haveno Functional Requirements
+
+This document maintains functional requirements for the Haveno user interface (work in progress).
+
+### Startup
+- On first startup, the user must configure a password.
+- On first startup, the user can use a local node or configure a remote node (url, username, password).
+- On startup, the application presents a splash screen with progress and description of startup state.
+
+### Wallet
+- The user can deposit into their Haveno wallet.
+- The user can generate new deposit subaddresses for their Haveno wallet.
```

Comment:
or a new one is generated for them?

Reply:
A new one should be generated automatically, and we can allow them to manually generate a new one.